UnicornRender/video/include/unicorn/video/Primitives.hpp

```diff
@@ -21,16 +21,27 @@ class Primitives
     /**
     *  @brief Fills given mesh with cube data
     *
-    *  @param  mesh    geometry mesh
+    *  @param[in] mesh    geometry mesh
     */
     UNICORN_EXPORT static void Cube(Mesh& mesh);
 
     /**
     *  @brief Fills given mesh with quad data
     *
-    *  @param  mesh    geometry mesh
+    *  @param[in] mesh    geometry mesh
     */
     UNICORN_EXPORT static void Quad(Mesh& mesh);
+
+    /**
+    *  @brief Fills given mesh with sphere data
+    *
+    *  @param[in] mesh geometry mesh
+    *  @param[in] radius radius of sphere
+    *  @param[in] rings horizontal slices count
+    *  @param[in] sectors vertical slices count
+    *  @attention rings * sectors must be higher then 16, less then std::vector::max_size() and less then sizeof(uint32_t)
```

Comment:
```
@attention `rings * sectors` must be greater than 16 and less than `sizeof(uint32_t)`
```

I would expect that `std::vector::max_size()` is greater or equal to sizeof(uint32_t)